templates/globals/updatable_profile_image.html

```diff
@@ -0,0 +1,25 @@
+{% load static %}
+
+<style>
+    {% include 'globals/custom.css' %}
+</style>
+
+<a tabindex="0" class="btn rounded-circle p-1" id="profile-image-popover" role="button" data-toggle="popover">
```

Comment:
Put `id` before `class` attribute and and a style `cursor: pointer;` (notice the space after `:` and `;` at end.